lib/seattleflu/id3c/cli/command/reportable_conditions.py

```diff
@@ -79,7 +73,7 @@ def webhook(suffix):
 
     reportable_conditions = db.cursor("reportable_conditions")
     reportable_conditions.execute("""
-        select reportable_condition_v1.*, presence_absence_id as id
+        select reportable_hcov19_v1.*, presence_absence_id as id
             from shipping.reportable_condition_v1
```

Comment:
The select-list is using the new view name but the from clause is using the old view name (although in my other comments I think it should stay that way). This generates a SQL error from Pg.